toolboxes/scripts/VisTools.py

```diff
@@ -745,6 +745,23 @@ def execute(self, parameters, messages):
         outputTargets = parameters[8].valueAsText # 8 - Output Targets
         inputObstructionFeatures = parameters[9].valueAsText # 9 - Input Obstruction Features - optional
 
+        # Check if a valid observer is supplied
```

Comment:
This looks good - my only recommendation is that this be moved into [VisibilityUtilities.linearLineOfSight]( https://github.com/Esri/military-tools-geoprocessing-toolbox/blob/dev/toolboxes/scripts/VisibilityUtilities.py#L1148) just in case we need to keep using the .tbx version (due to the slow open time in ArcMap) - that would also keep all of the processing in that method and keep this execute method lightweight